pkgs/tools/networking/snabb/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchgit}:
+
+stdenv.mkDerivation rec {
+  name = "snabb-v2015.10";
+
+  src = fetchgit {
+    url = "git://github.com/SnabbCo/snabbswitch";
+    rev = "20532eff595fef4bf1575d70e7963060a91add45";
```

Comment:
Actually best to use https://github.com/SnabbCo/snabbswitch/archive/v2015.10.tar.gz (we prefer releases over git revisions)